example-overrides/local-overrides-osp16-2-with-dpdk.yaml

```diff
@@ -1,9 +0,0 @@
-dpdk_interface: enp130s0f0
```

Comment:
why is this example being removed?

Reply:
it's renamed for 17.1. No point of duplicating examples for DPDK.